scripts/shared.js

```diff
@@ -12,4 +12,8 @@ export async function sleep(ms) {
     return new Promise((resolve) => {
         setTimeout(resolve, ms);
     })
-}
\ No newline at end of file
+}
+
+export const resources = [
```

Comment:
Preferably we shouldn't be specifying what resources to compile.

Instead I suggest we take a look at the `.toml` file for the server and extract resources that way.

If we need to ignore certain folders from compiling maybe consider adding a file inside of a resource folder called `.nocompile` or `.ignore` or something of the sort and immediately skip that resource entirely if it is present.